metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -634,6 +635,23 @@ def _convert_value(param):
         bold=True,
     )
 
+    workflow_url = (
+        "%s/workflows/%s/%s"
+        % (
+            ARGO_WORKFLOWS_UI_URL.rstrip("/"),
+            KUBERNETES_NAMESPACE,
+            argo_workflow_id,
+        )
+        if ARGO_WORKFLOWS_UI_URL
+        else None
+    )
+
+    if workflow_url:
+        obj.echo(
+            "See it in the Argo Workflows UI at %s" % workflow_url,
```

Comment:
@amerberg, we could perhaps change the string a bit so that the overall output is less confusing - 

```
See the run in the UI at xxx
and in the Argo Workflows UI at yyy
```

Reply:
I've updated to use that wording, with separate cases for when only one of the URLs is available.